src/ui/components/shared/PortalDropdown.tsx

```diff
@@ -66,7 +58,7 @@ export default function PortalDropdown(props: PortalDropdownProps) {
               <div
                 className={`content ${position}`}
                 style={{ ...style, ...contentPosition }}
-                ref={dropdownRef}
+                ref={setDropdownNode}
```

Comment:
The ref-setter was just passing through to the state-updater, so I just used it directly.